.github/workflows/test.yml

```diff
@@ -360,3 +359,48 @@ jobs:
           $env:PATH = "C:\msys64\usr\bin;$env:PATH"
           & "C:\msys64\mingw64\bin\mingw32-make.exe" -C $(Convert-Path test\old\testdir) VERBOSE=1
           $env:PATH = $OldPath
+
+  with-external-deps:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: |
+          sudo add-apt-repository ppa:neovim-ppa/stable
+          ./.github/scripts/install_deps.sh
+          sudo apt-get install -y \
+            libluajit-5.1-dev \
+            libmsgpack-dev \
+            libtermkey-dev \
+            libtree-sitter-dev \
+            libunibilium-dev \
+            libuv1-dev \
+            lua-busted \
+            lua-filesystem \
+            lua-inspect \
+            lua-lpeg \
+            lua-nvim \
```

Comment:
Is this the lua-client? Is this still needed after https://github.com/neovim/neovim/pull/22077 ?

Reply:
Yes, it's the lua client. Not needed, but one of its dependencies, mpack, is needed. Removing nvim client and explicitly install mpack instead.